store/mongo/datastore_mongo.go

```diff
@@ -413,17 +426,17 @@ func (db *DataStoreMongo) GetAllJobs(
 	sortField := bson.M{}
 	sortField["insert_time"] = -1
 	findOptions.SetSort(sortField)
-	cur, err := collection.Find(ctx, bson.M{}, findOptions)
+	cur, _ := collection.Find(ctx, bson.M{}, findOptions)
```

Comment:
We should return the error, not swallowing it:
```suggestion
      cur, err := collection.Find(ctx, bson.M{}, findOptions)
      if err != nil {
              return []model.Job{}, 0, err
      }
```